2020-02-Spring/2_linked_lists/README.md

```diff
@@ -126,31 +147,117 @@ Go to [Top](#top)
 <!-- Don't remove -->
 <a name="s2"/>
 
-### 2. SOLUTION 2 TODO :bug:
-
-Source: TODO :bug:
+### 2. Remove Zero Sum Consecutive Nodes from Linked List
 
 #### Naive/Simple Solution
+The idea is to locate the sublist of nodes that sum to 0 and de-list them.<br>
+- Starting at the edging node, find lower-bound and upper-bound node of the sublist that sum to 0.
+- if found, de-list nodes from [LBound, UBound], including by re-link PreLBound->next = UBound->next;
```

Comment:
A few suggestions:

* Recommend using monospacing for all complexity estimations and inline pseudocode. 
* Be consistent with capitalizing the start of a bullet -- either do it always or never do it.
* Explicitly mention that the bounds are inclusive; _I_ realize that you're using that notation to indicate inclusive bounds, but not everyone will understand that.
* Use phrasing which is more technical than "de-list" and "including by re-link".
* Make the second half of your sentence a complete clause.

```suggestion
- If found, remove nodes from `[LBound, UBound]` inclusive, and be sure to reset the preceding node's pointer, e.g. `PreLBound->next = UBound->next`;
```